src/components/profile/profile-settings/profileSettings.module.scss

```diff
@@ -0,0 +1,142 @@
+@import '@/src/styles/mixins';
+@import '@/src/styles/colors';
+
+
+@mixin wh($w, $h) {
+  width: $w;
+  height: $h;
+}
+
+.profile {
+  height: auto;
+}
+
+.navMenu {
+  @include wh(726px, 36px);
+  display: flex;
+  justify-content: space-between;
+  padding: 0;
+  list-style-type: none;
+  text-align: center;
+  margin: 36px 310px 0 13px;
+}
+
+li {
+  @include wh(181.5px, 36px);
+  display: flex;
+  justify-content: center;
+  color: $color-dark-300;
+  border-bottom: 1px solid $color-dark-300;
+
+  :hover {
+    @include wh(181.5px, 36px);
+    display: flex;
+    justify-content: center;
+    color: $color-accent-500;
+    border-bottom: 1px solid $color-accent-500;
+  }
+}
+
+.link {
+  color: $color-dark-300;
+  text-decoration: none;
+}
+
+.content {
+  display: flex;
+  flex-direction: column;
+  margin-left: 40px;
+  width: 726px;
+}
+
+.userInfo {
+  display: flex;
```

Comment:
for flex in your sass files  use base mixin from src/styles/mixins

Reply:
done